conn.go

```diff
@@ -261,7 +267,8 @@ func (c *Conn) start() { //nolint gocognit
 	for {
 		n, _, src, err := c.socket.ReadFrom(b)
 		if err != nil {
-			return
+			c.log.Warnf("Failed to ReadFrom %q %v", src, err)
+			continue
```

Comment:
one error would cause this whole process to fail from one potentially bad packet from any client on the interface